src/dotnet/APIView/APIViewWeb/Models/AssemblyModel.cs

```diff
@@ -47,11 +46,11 @@ public AssemblyModel(Stream stream, string fileName)
             AssemblyAPIV assembly = AssemblyAPIV.AssemblyFromStream(stream);
             if (assembly == null)
```

Comment:
Does this ever happen anymore?

Reply:
Initializing from a stream does - using a filepath is outdated now, though. Updated!

Reply:
> Initializing from a stream does 

In what case?